sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/config/LocalThroughputControlGroup.java

```diff
@@ -12,8 +13,9 @@ public LocalThroughputControlGroup(
         CosmosAsyncContainer targetContainer,
         Integer targetThroughput,
         Double targetThroughputThreshold,
+        PriorityLevel priorityLevel,
```

Comment:
I think having a separate PriorityThroughputControlGroup will be more clear for the intention

Reply:
will close this comment for now, as I think all the questions related to how we want to integrate client side throughput control with server side throughput control